tpm2/test/tpm2_test.go

```diff
@@ -32,6 +33,7 @@ import (
 	"testing"
 
 	"github.com/google/go-tpm-tools/simulator"
+	"github.com/google/go-tpm/tpm2"
 	. "github.com/google/go-tpm/tpm2"
```

Comment:
We shouldn't be including this package twice, we should only use the `.` import not the named import.